src/components/playback/mediasession.js

```diff
@@ -1,45 +1,26 @@
-define(['playbackManager', 'nowPlayingHelper', 'events', 'connectionManager'], function (playbackManager, nowPlayingHelper, events, connectionManager) {
-    "use strict";
-
-    // no support for mediaSession
-    if (!navigator.mediaSession && !window.NativeShell) {
-        return;
-    }
+import playbackManager from 'playbackManager';
+import nowPlayingHelper from 'nowPlayingHelper';
+import events from 'events';
+import connectionManager from 'connectionManager';
+/* eslint-disable indent */
 
     // Reports media playback to the device for lock screen control
 
-    var currentPlayer;
-    var lastUpdateTime = 0;
-
-    function seriesImageUrl(item, options) {
+    let currentPlayer;
 
+    function seriesImageUrl(item, options = {}, type = options.type || 'Primary') {
         if (item.Type !== 'Episode') {
```

Comment:
```suggestion
        options = options || {};
        options.type = options.type || "Primary";
        if (item.Type !== 'Episode') {
```

Adding this back fixes what I commented in my review, as pointed out by @JustAMan.

Reply:
The default for "options" is set in line 11.

The optional "type" argument, which is new, default to either the value of options.type or "Primary", as defaults for arguments have access to all of the previous argument's values.

Reply:
Oh, I see what you mean.